packages/ui-components/src/components/select/select.tsx

```diff
@@ -17,64 +16,41 @@ export class KvSelect implements ISelect, ISelectEvents {
 	/** @inheritdoc */
 	@Prop({ reflect: true }) searchValue?: string;
 	/** @inheritdoc */
-	@Prop({ reflect: true }) selectionClearable?: boolean = false;
-	/** @inheritdoc */
 	@Prop({ reflect: true }) searchPlaceholder?: string;
 	/** @inheritdoc */
-	@Prop({ reflect: true }) selectionClearEnabled?: boolean;
-	/** @inheritdoc */
-	@Prop({ reflect: true }) clearSelectionLabel?: string = SELECT_CLEAR_SELECTION_LABEL;
-	/** @inheritdoc */
 	@Prop({ reflect: true }) minHeight?: string;
 	/** @inheritdoc */
 	@Prop({ reflect: true }) maxHeight?: string;
+	/** @inheritdoc */
+	@Prop({ reflect: true }) hasHeaderActions?: boolean = false;
 
 	/** @inheritdoc */
 	@Event() searchChange: EventEmitter<string>;
-	/** @inheritdoc */
-	@Event() clearSelection: EventEmitter<void>;
 
 	private onSearchChange = (event: CustomEvent<string>) => {
 		this.searchChange.emit(event.detail);
 	};
 
-	private onClearSelection = () => {
-		this.clearSelection.emit();
-	};
-
-	private getCustomStyle = () =>
-		omitBy(
+	private get customStyle() {
```

Comment:
Any particular reason to change this to a get? Or is it just a syntax preference?

Reply:
just syntax preference, can revert if you prefer :D